source/ptreferential/tests/ptref_test.cpp

```diff
@@ -698,3 +700,84 @@ BOOST_AUTO_TEST_CASE(code_request) {
     BOOST_REQUIRE_EQUAL(res.size(), 1);
     BOOST_CHECK_EQUAL(b.data->pt_data->stop_areas.at(res.at(0))->uri, "A");
 }
+
+BOOST_AUTO_TEST_CASE(contributor_and_frame) {
+
+    ed::builder b("201601011T1739");
```

Comment:
If the default frames and contributor are only used in one test, I think it would be clearer to add them only there no ?